internal/abci/abci.go

```diff
@@ -43,6 +45,9 @@ type Chain interface {
 }
 
 type State interface {
+	// ChainID returns the ID of the network's chain
+	ChainID() (string, error)
```

Comment:
I would recommend changing chainId to network id to avoid confusion between what tendermint thinks is a chain and what accumulate chains are.

Reply:
I changed it to `SubnetID`, since BVNs and the DN are subnetworks of the Accumulate network. WDYT?

Reply:
I think that is good.